tools/gather_logs_docker.sh

```diff
@@ -0,0 +1,23 @@
+rm -rf ovn-control-plane
+rm -rf ovn-worker
+rm -rf ovn-worker2
+mkdir -p ovn-control-plane
+mkdir -p ovn-worker
+mkdir -p ovn-worker2
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump.pcap .
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump_logs.log .
+kubectl --kubeconfig=/home/alexey/admin.conf get pods --all-namespaces -o wide > kube-topology
+for d in ovn-control-plane ovn-worker ovn-worker2
+do
+		    docker cp $d:/var/log/openvswitch/ovn-controller.log $d/.  
+			    docker cp $d:/var/log/openvswitch/ovs-vswitchd.log $d/.     
+				    docker cp $d:/var/log/openvswitch/ovsdb-server.log $d/.  
+					    docker cp $d:/var/log/ovn-kubernetes/ovn-k8s-cni-overlay.log $d/.
+						        docker cp $d:/var/log/ovn-kubernetes/ovnkube.log $d/.
+						done
```

Comment:
I'd suggest to try to copy these files from all nodes, for the case when system runs in the HA mode.
May be we can send warning/error messages to /dev/null

Reply:
in `HA` mode there are three nodes:
```bash
NAME                STATUS     ROLES                  AGE   VERSION
ovn-control-plane   NotReady   control-plane,master   54m   v1.20.0
ovn-worker          NotReady   master                 53m   v1.20.0
ovn-worker2         NotReady   master                 53m   v1.20.0
```
those parts are part of the following for loop :
```bash
for d in ovn-control-plane ovn-worker ovn-worker2
do
    docker cp $d:/var/log/openvswitch/ovn-controller.log $d/.  
    docker cp $d:/var/log/openvswitch/ovs-vswitchd.log $d/.         
    docker cp $d:/var/log/openvswitch/ovsdb-server.log $d/.  
    docker cp $d:/var/log/ovn-kubernetes/ovn-k8s-cni-overlay.log $d/.
    docker cp $d:/var/log/ovn-kubernetes/ovnkube.log $d/.
done     
```
so as far as I understand we already copying those files from all the nodes.